commands/addvertical.js

```diff
@@ -71,11 +77,53 @@ class VerticalAdder {
           required: true,
           type: 'singleoption',
           options: this._getPageTemplates(jamboConfig)
+        },
+        locales: {
+          displayName: 'Additional Page Locales',
+          type: 'multioption',
+          options: this._getAdditionalPageLocales(jamboConfig)
         }
       }
     };
   }
 
+  /**
+   * @returns {Array<string>} The additional locales that are configured in 
+   *                          locale_config.json
+   */
+  static _getAdditionalPageLocales(jamboConfig) {
+    if (!jamboConfig) {
+      return [];
+    }
+
+    const configDir = jamboConfig.dirs.config;
+    if (!configDir) {
+      return [];
+    }
+
+    const localeConfig = path.resolve(configDir, 'locale_config.json');
+    if (!fs.existsSync(localeConfig)) {
+      return [];
+    }
+
+    const pageLocales = [];
+    const localeContentsRaw = fs.readFileSync(localeConfig, 'utf-8');
+    let localeContentsJson;
+    try {
+      localeContentsJson = parse(localeContentsRaw);
+    } catch(err) {
+      throw new UserError('Could not parse locale_config.json ', err.stack);
+    }
+    const defaultLocale = localeContentsJson.default;
+    for (const locale in localeContentsJson.localeConfig) {
```

Comment:
nit: Can you declare `pageLocales` above the for loop since it's not needed until the loop? Alternatively, you could remove `pageLocales` and use an array filter